src/main/java/org/broadinstitute/hellbender/engine/filters/CountingReadFilter.java

```diff
@@ -219,6 +220,48 @@ public boolean test(final GATKRead read) {
             return accept;
         }
 
+        @Override
+        protected String getSummaryLineForLevel(final int indentLevel) {
+            if (0 == filteredCount) {
+                return "No reads filtered by: " + getName();
```

Comment:
(If you do take my suggestion to swap the order of the checks, you'll need to remove the `else` in order to handle the case where simplified output can't be generated due to ORs/NOTs, and 0 total reads were filtered at the root level)

Reply:
I agree with your suggestion.  I just pushed a change that will include that.  As part of making the change, I also made a minor change to the way summary output is printed for all types of CountingReadFilter to make it consistent (basically "0 read(s)" instead of "No reads"), and updated the existing test cases accordingly (after first forgetting to do so).